src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState(false);
  const [isLoading, setIsLoading] = useState(false);
```
Name it the same as you did above, so your boolean values have the same naming rule